prisma/schema.prisma

```diff
@@ -396,6 +396,22 @@ model Season {
   isOpenForRegistration Boolean
   isOpenForFSquared Boolean @default(false)
   registrationSize Int @default(60)
+
+  Weeks SeasonWeek[]
+}
+
+model SeasonWeek {
+  id String @id @default(cuid())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  season Season  @relation(fields: [seasonId], references: [id], onDelete: Cascade, onUpdate: Cascade)
+  seasonId String
+
+  weekNumber Int
+
+  weekStart DateTime
+  weekEnd DateTime
```

Comment:
should we also have a connection from SeasonWeek to NFLGame?

Reply:
At some point definitely, just was trying to keep the scope down even though this is already giant, ha.